src/uct/cuda/cuda_copy/cuda_copy_iface.c

```diff
@@ -134,7 +144,7 @@ static ucs_status_t uct_cuda_copy_iface_query(uct_iface_h tl_iface,
 
     iface_attr->latency                 = UCT_CUDA_COPY_IFACE_LATENCY;
     iface_attr->bandwidth.dedicated     = 0;
-    iface_attr->bandwidth.shared        = iface->config.bandwidth;
+    iface_attr->bandwidth.shared        = iface->config.bw.other;
```

Comment:
maybe we can use just constant here to preserve wire compatibility and avoid adding `other` key?
@yosefe, wdyt?